internal/app/mistapiconnector/stats_collector.go

```diff
@@ -98,11 +98,11 @@ func createMetricsEvent(nodeID, region string, info *streamInfo, metrics *stream
 				MediaTimeMs: push.Stats.MediaTime,
 			}
 			if metrics.Bytes > pushInfo.pushedBytes {
-				census.IncMultistreamBytes(metrics.Bytes - pushInfo.pushedBytes)
+				census.IncMultistreamBytes(metrics.Bytes-pushInfo.pushedBytes, info.stream.ID)
```

Comment:
I think we should use `info.stream.PlaybackID` (which we call manifestID in other places).
@victorges @iameli wdyt?

Reply:
is `PlaybackID`/`manifestID` the ID that appears in the livepeer.com dashboard? ie. `ad73b342-2c64-4a51-af55-47e830ba1d05`

Reply:
stream.ID it is in dashboard:
<img width="438" alt="image" src="https://user-images.githubusercontent.com/2035357/153038410-9d8faf74-8c10-486d-a72a-38bf6f24d2e1.png">
stream.PlaybackID - it is part of Playback URL:
![image](https://user-images.githubusercontent.com/2035357/153038588-344fa43e-d7f1-4bfa-a652-784905f6270c.png)
(https://cdn.livepeer.monster/hls/ef6aah6kn7zzdaxd/index.m3u8)
And stream.PlaybackID in our logs gets logged as `manifestID`

Reply:
And most log lines contains `manifestID`, no `stream.ID`